ДЗ. Верстка главной страницы

VladislavShtefan opened the pull request:
ДЗ. Верстка

VladislavShtefan committed:
ДЗ. Верстка главной страницы

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+    
+    <!-- HEADER -->
+    <header>
+        <nav class="top_menu">
+            <a class="nav_logo" href="#"><img src="img/logo.png" alt="#"></a>
+              <a class="nav_link_top" href="#">BLOG</a>
+              <a class="nav_link_top" href="#">CATALOG</a>
+              <a class="nav_link_top" href="#">ABOUT US</a>
+        </nav>
+        <nav class="bottom_menu">
+          <a class="nav_search" href="#"><img src="img/search.png" alt="#"></a>
+              <a class="nav_link_bottom" href="#">CONTACT US</a>
+        </nav>
```
Если у тебя идёт несколько элементов, используй список

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+    <main class="main">
+       <div class="main_article">
+            <div class="main_top">
+                <H2>01</H2>
```
01 не является заголовком, переключатели ещё нужно добавить

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+            <ul>
+                <li>
+                    <img src="img/product1.png" alt="#">
+                    <p><a href="#">APPAREL KIDS</a></p>
```
оборачивать в тег p не нужно, оставь просто ссылку

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+            </ul>
+        </div>
+        <div class="product_title">
+            <img  src="img/arrow_left.png" alt="#">
+            <img src="img/arrow_right.png" alt="#">
```
Это не картинки, а кнопки. Выполняют функцию пролистывания категорий

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+        <div class="card">
+                <img src="img/lady_id_dress.svg" alt="lady">
+            <div class="container_card">
+                <h2>Delivery</h2>
+                <p><b><i>Delivery</i></b></p>
+                <p><i>Guarantee</i></p>
+                <p><i>Credit</i></p>
+                <p>Some days i feel like i've been working on dlk my entire life and there are other days that i am completely shocked that four years have flown by. It's simply magical.</p>
+                <p>What started as an outlet to deal  with the traumatic loss of my husband, dlk has now become a truly happy place for me and i have you and all of the beautiful designers that we work with to thank!</p>
+                <button type="button">LEARN MORE</button>
```
Тут тоже перечисление элементов, используй списки

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+                <img src="img/lady_id_dress.svg" alt="lady">
+            <div class="container_card">
+                <h2>Delivery</h2>
+                <p><b><i>Delivery</i></b></p>
+                <p><i>Guarantee</i></p>
+                <p><i>Credit</i></p>
```
Стилизацию через теги не делай, потом в css сделаешь

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+
+
+            <!-- Sponsor -->
+            <div class="sponsor">
```
Тут не див, а секция и заголовок нужно прописать

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+
+            <!-- Card -->
+            <div class="container_contact_us">
+                <div class="card_about_us">
```
не див, а секция

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+                    <button type="button">buy now</button>
+                </div>
+
+                <div class="card_map">
```
Тоже самое, тут контент как самостоятельная часть. Нужно заменить див на секцию

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,169 @@
+            <div class="footer">
+            
+                <section>
+                    <h5>Navigation</h5>
```
Не ломай последовательность заголовков. В каждой след секции исчисление заголовков начинается с h2

VladislavShtefan committed:
ДЗ исправил

Yan1993-Dev commented on index.html:
```diff
@@ -28,7 +30,11 @@
        <div class="main_article">
             <div class="main_top">
-                <H2>01</H2>
+                <p>01</p>
+                <button type="button"><img src="img/Rectangle1.png" alt="#"></button>
+                <button type="button"><img src="img/Rectangle.png" alt="#"></button>
+                <button type="button"><img src="img/Rectangle.png" alt="#"></button>
+                <button type="button"><img src="img/Rectangle.png" alt="#"></button>
```
Когда идёт перечисление элементов, используй списки

Yan1993-Dev commented on index.html:
```diff
@@ -28,7 +30,11 @@
     <main class="main">
        <div class="main_article">
```
Странный класс, по сути это слайдер. Можно переименовать в slider

Yan1993-Dev commented on index.html:
```diff
@@ -28,7 +30,11 @@
     <main class="main">
        <div class="main_article">
             <div class="main_top">
```
main_top -> slider__controls

Yan1993-Dev commented on index.html:
```diff
@@ -72,49 +78,50 @@ <h2>Category Products</h2>
             <!-- Sponsor -->
-            <div class="sponsor">
+            <section class="sponsor">
+                <h2>#</h2>
```
Текст в заголовках указывать нужно, например sponsors. Потом скроем

VladislavShtefan committed:
ДЗ сделал

Полностью переписал главную страницу, написал классы, вставил картинки, вставил ссылки в некоторых кнопках и логотипе. И просто привел код в порядок. Посмотри пожалуйста. Спасибо!

Yan1993-Dev commented on index.html:
```diff
@@ -2,175 +2,366 @@
-              <li class="nav_link_bottom">CONTACT US</li>
-            </ul>
-        </nav>
+    <header class="nav">
```
nav обычно указывают класс навигации. Тут у тебя блок header. Лучше переименовать, ну и ниже сделать так же

Yan1993-Dev commented on index.html:
```diff
@@ -2,175 +2,366 @@
+                </a>
+            </li>
+            <li class="nav__list-item">
+                <a class="nav__list-link" href="index.html">BLOG</a>
+            </li>
+            <li class="nav__list-item">
+                <a class="nav__list-link" href="index.html">CATALOG</a>
+            </li>
+            <li class="nav__list-item">
+                <a class="nav__list-link" href="index.html">ABOUT US</a>
```
Ну все ссылки на индекс делать не нужно, пока оставил бы с #

Yan1993-Dev commented on index.html:
```diff
@@ -2,175 +2,366 @@
-                <p>Designer in Slovenia</p>
-                <button type="button">EXPORE NOW</button>
-                <img src="img/main.png" alt="#">
+        <section class="main__banner">
```
тут можно убрать приставку main, а оставить только banner

Yan1993-Dev commented on index.html:
```diff
@@ -2,175 +2,366 @@
+            <div class="banner__title">
+                <ul class="banner__list-title">
+                    <li class="banner__list-item">
+                        <h1 class="banner__list-text">THE OOH NOO COLLECTION</h1>
```
banner__list-text -> banner__list-title

Yan1993-Dev commented on index.html:
```diff
@@ -2,175 +2,366 @@
-            
+         <section class="delivery__article">
+            <div class="delivery__img">
+                <img src="img/lady_id_dress.svg" alt="lady__in__dress">
```
lady_id_dress - > lady-in-dress
Лучше именовать картинки с несколькими словами через -

VladislavShtefan committed:
Главная ДЗ

Правки +
